launcher/minecraft/skins/SkinModel.cpp

```diff
@@ -23,10 +23,17 @@
 #include "FileSystem.h"
 #include "Json.h"
 
-static QImage improveSkin(const QImage& skin)
+static QImage improveSkin(const QImage& _skin)
 {
+    auto skin = _skin;
+    // It seems some older skins may use this format, which can't be drawn onto
+    // https://github.com/PrismLauncher/PrismLauncher/issues/4032
+    // https://doc.qt.io/qt-6/qpainter.html#begin
+    if (skin.format() != QImage::Format_RGBA8888) {
```

Comment:
```suggestion
    if (skin.format() == QImage::Format_Indexed8 {
```
This is wrong. See the previous review